DOCS/new-license-workflow.md

```diff
@@ -17,23 +24,154 @@ Ask the submitter for any additional info needed, preferably via the Github issu
+```
+1. Definitions
+
+    a. “IGO” means, solely and exclusively for purposes of this License, an organization established by a treaty or other instrument governed by international law and possessing its own international legal personality. Other organizations established to carry out activities across national borders and that accordingly enjoy immunity from legal process are also IGOs for the sole and exclusive purposes of this License. IGOs may include as members, in addition to states, other entities.
+```
+4. Save the file and [commit it](https://docs.github.com/en/github/committing-changes-to-your-project) to your clone of the repository.
+
+That's it! The test .txt file is now ready to go and you can move on to the XML file. It's slightly trickier than the test .txt file, but not much (at least not in the basic case that we cover here).
+
+### Add the XML file
+
+1. The license XML files are in the `src` directory. If the license you're working on does not yet have an XML file there, find the XML file for a similar license then copy it to be the basis of the new license XML file. For instance, for the _K-9 Robotic Dog Hardware License_ (henceforth referred to by its licenseId), you could copy the _CERN Open Hardware License v1.1_ file, `CERN-OHL-1.1.xml` to `K-9RDHL.xml`, then go from there.
+1. Set the attributes in the `license` tag:
+    * licenseId: Set to the licenseId for that license. This must match the licenseId used for to name both the XML and test .txt files.
+    * name: The canonical name of the license. This must match the name of the license in the test .txt file. The value set in this attribute is what will appear in the [SPDX License List](https://spdx.org/licenses/).
+    * isOsiApproved: Set to `true` if the license has been [approved by the Open Source Initiative](https://opensource.org/licenses) and `false` else.
+    * listVersionAdded: Set to the [SPDX License List Version](https://github.com/spdx/license-list-XML/milestones) in which this license is being added. This version is set in the Milestone assigned to the issue.
+    * If you copied the XML file of another license, remove any attributes used for that license but not needed for this new one. 
+1. Set the value of the `crossRef` tag to the URL for the canonical version of the license. 
+    * This should be the same as the URL used for the text added to the test .txt file above.
+    * If the XML file you copied includes any additional `crossRef` tags, make sure you remove them.
+1. Add the text of the license in the `text` tag. There are several special things to pay attention to here:
```

Comment:
I'll also add an item here noting the optional use of the `notes` tag, for explanatory notes that might be helpful to users.